### DISCUSSION TITLE
Issues with pixelation_shader

### DISCUSSION THREAD

**Discussion by JacktheFabricRipper** [pokepetter/ursina#269] [Q&A]
I'm making a simple dungeon game, and I'm trying to use pixelation_shader, but whenever I do, it gives me the following error

Traceback (most recent call last):
  File "C:\Users\-----\OneDrive\Desktop\dungeon game\dungeon_game.py", line 3, in <module>
    from ursina.shaders import pixelation_shader
ImportError: cannot import name 'pixelation_shader' from 'ursina.shaders' (C:\Users\18019\AppData\Local\Programs\Python\Python39\lib\site-packages\ursina\shaders\__init__.py)

If I use other shaders it works, so I know its not formatting. Does anyone know the issue?

**Comment by PixelTomato** [comment]
The pixilation shader is a screen-space shader and as such you will need to import it like so:
```py
from ursina.shaders.screenspace_shaders import pixelation_shader
camera.shader = pixelation_shader.pixelation_shader
```

**Reply by JacktheFabricRipper** [reply]
So I tried that and I got the following error:

    from ursina.shaders.screenspace_shaders import pixelation_shader
ImportError: cannot import name 'pixelation_shader' from 'ursina.shaders.screenspace_shaders' (unknown location)
>>>

**Comment by pokepetter** [comment]
I just forgot to add it to the list. You should still be able to do from ursina.shaders.screenspace_shaders.pixelation_shader import pixelation_shader though.

**Reply by JacktheFabricRipper** [reply]
thanks! I'll try that real quick

**Reply by JacktheFabricRipper** [reply]
So this happened...   

 from ursina.shaders.screenspace_shaders.pixelation_shader import pixelation_shader
ModuleNotFoundError: No module named 'ursina.shaders.screenspace_shaders.pixelation_shader'

**Reply by pokepetter** [reply]
Try updating, sounds like you have an old version.

**Reply by JacktheFabricRipper** [reply]
also is it just me or does the screen just gain a red tint whenever you add a screenspace shader

**Reply by pokepetter** [reply]
That means there's an error. Either in the shader program or a lack of supported gpu or opengl version.

**Comment by JacktheFabricRipper** [comment]
Traceback (most recent call last):
  File "C:\Users\18019\OneDrive\Desktop\Randomly Generated Worlds\Randomly Generated World.py", line 8, in <module>
    from ursina.shaders.screenspace_shaders.pixelation_shader import pixelation_shader
ModuleNotFoundError: No module named 'ursina.shaders.screenspace_shaders.pixelation_shader'
